packages/api-server/api_server/routes/tasks/scheduled_tasks.py

```diff
@@ -159,6 +160,75 @@ async def del_scheduled_tasks_event(
     await schedule_task(task, task_repo)
 
 
+@router.post(
+    "/{task_id}/update", status_code=201, response_model=ttm.ScheduledTaskPydantic
+)
+async def update_schedule_task(
+    task_id: int,
+    scheduled_task_request: PostScheduledTaskRequest,
+    except_date: Optional[datetime] = None,
+    task_repo: TaskRepository = Depends(task_repo_dep),
+):
+    try:
+        task = await get_scheduled_task(task_id)
+        if task is None:
+            raise HTTPException(404)
+
+        # If except_date; single event: that event is deleted from the previous task, and a new one is created
+        #   with the data requested in the schedule form [scheduled_task_request.schedules].
+        # Else; entire series: Task is completely deleted and another one is created
+        #   with the changes requested by the user.
+        async with tortoise.transactions.in_transaction():
+            if except_date:
```

Comment:
Perhaps have the comments above this `if` and `else` case to make it easier to follow